tournament/schedule.go

```diff
@@ -44,7 +44,7 @@ func BuildRandomSchedule(teams []model.Team, scheduleBlocks []model.ScheduleBloc
 	if err != nil {
 		return nil, err
 	}
-	if len(csvLines) != numMatches {
+	if len(csvLines) < numMatches {
```

Comment:
This checks whether there are enough matches in the match file (as above).  It's exactly equal in the default mode where there are dozens of different possible schedules, but in our mode (where we're just loading a pair of fixed files, since we're only doing the 12 and 13 way matches), I just make sure there are enough matches in the file to continue.